packages/core/src/plugins/svgo.js

```diff
@@ -26,7 +36,7 @@ export default async (code, config = {}, state = {}) => {
   const filePath = state.filePath || process.cwd()
   const svgoRcConfig = await explorer.search(filePath)
   const svgo = new SVGO(
-    merge(getBaseSvgoConfig(config), svgoRcConfig, config.svgoConfig),
+    mergeWith(getBaseSvgoConfig(config), svgoRcConfig, config.svgoConfig, concatArrays),
```

Comment:
Nitpick: like suggested, I would extract the "merging" into a separate function and test that in isolation to make sure the "merged" config is correct.